moveit_core/trajectory_processing/src/ruckig_traj_smoothing.cpp

```diff
@@ -220,7 +220,7 @@ bool RuckigSmoothing::runRuckig(robot_trajectory::RobotTrajectory& trajectory,
   trajectory.unwind();
 
   // Initialize the smoother
-  double timestep = trajectory.getAverageSegmentDuration();
+  double timestep = robot_trajectory::average_segment_duration(trajectory).seconds();
```

Comment:
Why not use the provided member function instead? We shouldn't mix these two different paradigms. If you are planning to make RobotTrajectory a pure data type with free functions, we can talk about that. Here this change is unnecessarily confusing.

Reply:
The reason I picked these two functions and changed them to functions is that you can't deprecate a return type of a function with an overload of the same name, similar to the places where I added an `At` suffix or `Deque` suffix.